code/datums/trading/traders/food.dm

```diff
@@ -102,7 +102,6 @@
 							/obj/item/chems/food/fruit_slice                = TRADER_BLACKLIST,
 							/obj/item/chems/food/slice                      = TRADER_BLACKLIST_ALL,
 							/obj/item/chems/food/grown                      = TRADER_BLACKLIST_ALL,
-							/obj/item/chems/food/human                      = TRADER_BLACKLIST_ALL,
```

Comment:
Feels like this should be `/obj/item/chems/food/human/burger` which itself should just be `humanburger`.